app/test/unit/infer-comparison-branch-test.ts

```diff
@@ -35,29 +35,16 @@ function createTestGhRepo(
   defaultBranch: string | null = null,
   parent: GitHubRepository | null = null
 ) {
-  if (owner.indexOf('/') !== -1) {
-    throw new Error(
-      'Providing a slash in the repository name is no longer supported, please update your test'
-    )
-  }
```

Comment:
Wondering if this check is still needed. I notice we're removing it and not incorporating a check elsewhere.

Reply:
yeah, i removed this because we don't have this check in any other places where we name a repository (and removing it didn't break anything). it was added in 3fe0747133274f97d92c3a9249c62f57c22e0fd3, presumably out of an abundance of caution as i can't find any evidence in the git history that we were ever using a `/` in our repository names.